### PR TITLE
Fill dynamic p2p Applications deep dive

### PR DESCRIPTION
**Pull request by jkarni**

### CODE REVIEW

File: DEEP_DIVES/46-building-dynamic-p2p-applications.md [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+Messages should be able to be delivered to well-connected nodes in at most 1 second
+for topics with under 10 connected subscribers, and at most 5 for ~1000 connected
+subscribers
 
 ## Where to learn about it
 
-<List of talks, links to github issues, code, notes and so on>
+- Radicle website: https://radicle.xyz/
+- Radicle repo: https://github.com/radicle-dev/radicle
+- PolderCast: https://hal.inria.fr/hal-01555561/document
```
Comment (alanshaw) [inline DEEP_DIVES/46-building-dynamic-p2p-applications.md]: Have you seen/tried the experimental [gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) implementation? Might be worth linking here along with:

* https://github.com/libp2p/specs/tree/master/pubsub - original floodsub spec
* https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/episub.md - proposal for a different pubsub based on Epidemic Broadcast Trees
* https://github.com/libp2p/research-pubsub - more pubsub research

File: DEEP_DIVES/46-building-dynamic-p2p-applications.md [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+IPFS and its associated technologies are mostly *static*. Radicle is an attempt to build 
+a *dynamic* P2P web on top of IPFS; one which can host applications with the same abilities
+of the centralized servers of today to receive inputs from various people or sources. 
+
 
 ## Goal
 
-<Define what we would get if the Open Problem(s) are solved for this Deep Dive>
+We've encountered a few difficulties in making this work smoothly. IPFS pubsub in particular
+is quite slow. One big advantage for us would be to be able to control the network topology
```
Comment (alanshaw) [inline DEEP_DIVES/46-building-dynamic-p2p-applications.md]: Would you be able to elaborate on "quite slow" - what aspect in particular? e.g. signing and sending a message? receiving a message from a node that you're not directly connected to?

**Review by alanshaw — CHANGES_REQUESTED** [review]
❤️ This is super cool! Are there any other aspects of IPFS in relation to dynamic p2p apps you've had issues with that you'd like to discuss?

**Comment by alanshaw** [comment]
ping @jkarni

File: DEEP_DIVES/46-building-dynamic-p2p-applications.md (hunk 1 above)
Comment (jkarni) [reply DEEP_DIVES/46-building-dynamic-p2p-applications.md]: Done - thanks!

**Comment by jkarni** [comment]
Sorry for the delay - rebased and added requested changes.

**Review by alanshaw — APPROVED** [review]

**Pull request merged**